src/plone/restapi/tests/http-examples/users_filtered_by_groups.req

```diff
@@ -0,0 +1,3 @@
+GET /plone/@users?groups_filter=Reviewers,Administrators HTTP/1.1
```

Comment:
@nileshgulia1 @ksuess according to our naming convention we should use a dash in "group-filters".

https://plonerestapi.readthedocs.io/en/latest/conventions.html?highlight=parameters#naming-convention-for-attribute-names-in-uris

The way we pass lists/arrays in a URL parameter is also something that is not fully clear right now. I currently can't find the discussion, just this:

https://github.com/plone/plone.restapi/issues/45

@sneridagh @jackahl do you recall our discussion regarding passing arrays as URL param? That was in the GET vs POST discussion on the search endpoint, right?

Reply:
Renaming of attribute groups_filter to groups-filter done.
https://github.com/plone/plone.restapi/pull/1325/commits/bc42a349bb7740d7a61155d4f0c50c92a66c5c9f

Reply:
groups-filter parameter now expected with colon list separator.
https://github.com/plone/plone.restapi/pull/1325/commits/3a32cb809af2384f0af155e23f6a19014b4040f4